src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,11 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
```

Comment:
It makes sense to use a loop instead of repeating code.
The System.out.println() method automatically calls the toString() method.